blockchain/state_manager/src/lib.rs

```diff
@@ -55,8 +54,7 @@ type CidPair = (Cid, Cid);
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "PascalCase")]
 pub struct InvocResult {
-    #[serde(with = "unsigned_message::json")]
-    pub msg: UnsignedMessage,
+    pub msg: Message,
```

Comment:
Will this still be serialized correctly? I really wish we had some tests to verify the desired behavior.